ietf/doc/utils_search.py

```diff
@@ -143,14 +143,15 @@ def fill_in_document_table_attributes(docs, have_telechat_date=False):
     xed_by = RelatedDocument.objects.filter(target__name__in=list(rfc_aliases.values()),
                                             relationship__in=("obs", "updates")).select_related('target')
     rel_rfc_aliases = dict([ (a.document.id, re.sub(r"rfc(\d+)", r"RFC \1", a.name, flags=re.IGNORECASE)) for a in DocAlias.objects.filter(name__startswith="rfc", docs__id__in=[rel.source_id for rel in xed_by]) ])
+    l = []
     for rel in xed_by:
         d = doc_dict[rel.target.document.id]
         if rel.relationship_id == "obs":
             l = d.obsoleted_by_list
         elif rel.relationship_id == "updates":
             l = d.updated_by_list
         l.append(rel_rfc_aliases[rel.source_id])
-        l.sort()
+    l.sort(key=lambda x: int(re.sub(r"rfc\s*(\d+)", r"\1", x, flags=re.IGNORECASE)))
```

Comment:
```suggestion
```

Reply:
In case the UI suggest feature doesn't result in what I meant, the goal is to not have any sort statement inside or after the loop.